### PR TITLE
Optimizing imports and renaming variables

### PR DESCRIPTION
**Pull request by jgabrielfreitas**
Removing a lot of useless imports and renaming variables of type `Activity` named `context`

### CODE REVIEW

File: src/internal/java/com/jakewharton/u2020/ui/bugreport/BugReportLens.java [hunk 1]
```diff
@@ -27,53 +27,53 @@
...
-
-            @Override public void onError(Throwable e) {
-              Toast.makeText(context, "Couldn't attach the logs.", Toast.LENGTH_SHORT).show();
-              submitReport(report, null);
-            }
-
-            @Override public void onNext(File logs) {
-              submitReport(report, logs);
-            }
+                .subscribeOn(Schedulers.io())
```
Comment (tasomaniac) [inline src/internal/java/com/jakewharton/u2020/ui/bugreport/BugReportLens.java]: Are you sure about this reformatting? The formatting might be different in square's code style. The same with the imports.

Comment (jgabrielfreitas) [inline src/internal/java/com/jakewharton/u2020/ui/bugreport/BugReportLens.java]: Right. I fixed the formatting issue. But what is the problem with imports?

**Comment by jgabrielfreitas** [comment]
Right. I fixed the formatting issue. But what is the problem with imports?

**Comment by solenwin2** [comment]
Readme_edit

**Pull request closed without merging**